CMakeLists.txt

```diff
@@ -101,7 +101,7 @@ endif()
 set(plutil_rpath)
 if(CMAKE_SYSTEM_NAME STREQUAL Android OR CMAKE_SYSTEM_NAME STREQUAL Linux OR
     CMAKE_SYSTEM_NAME STREQUAL FreeBSD)
-  set(Foundation_RPATH -Xlinker;-rpath;-Xlinker;"\\\$\$ORIGIN")
+  set(Foundation_RPATH -Xlinker;-rpath;-Xlinker;"\\\$\$ORIGIN";-Xlinker;-rpath;-Xlinker;"$ORIGIN/../lib")
```

Comment:
This is needed in clients not in Foundation itself (Foundation is already in lib and $ORIGIN should work)